### ISSUE TITLE
Authentication error.

### ISSUE THREAD

**Issue by Hendriks046** [milo526/homebridge-tuya-web#346]
**Describe the bug**

My tuya devices stopped working all of the sudden. Yesterday they where working fine. But today none of them are responding. I have checked the log and this is what it says:

15-10-2021 11:06:02Tuya Web BridgeTuyaWebPlatformWARNINGCannot acquire token, waiting 65 seconds.
15-10-2021 11:07:07Tuya Web BridgeTuyaWebPlatformRetrying authentication after previous error.
15-10-2021 11:07:07Tuya Web BridgeTuyaWebPlatformERRORAuthentication error: you cannot auth exceed once in 60 seconds

**Steps To Reproduce**

Restart or reboot the homebridge.

**Plugin Version**
 
 1.0.2

 
 **Homebridge Config**
```json
 {
    "accessories": [],
    "platforms": [
        {
            "platform": "TuyaWebPlatform",
            "name": "TuyaWebPlatform",
            "options": {
                "username": "---------------",
                "password": "----------",
                "countryCode": "31",
                "platform": "tuya"
            },
            "defaults": [
                {
                    "id": "----------------------",
                    "device_type": "garage"
                }
            ]
        }
    ]
}

 ```

**Device Config**
```Getting credentials
Got credentials
{   'errorMsg': 'you cannot auth exceed once in 60 seconds',
    'responseStatus': 'error'}
Traceback (most recent call last):
  File "main.py", line 35, in <module>
    payload = {"accessToken": auth_response["access_token"]}
KeyError: 'access_token'
 ```

**Comment by milo526** [comment]
Please turn of your homebridge for 5 minutes and then reboot.
Make sure you only have one plugin installed for your Tuya devices.

**Comment by Hendriks046** [comment]
> Glad you solved it, please see #148 for your other issue.

Milo, please reopen the issue. The error keeps coming back.. i’m not able to fix it with a specific way. Sometimes it takes 1 time reboot. Sometimes i’m trying rebooting for hours…..
